lib/docs.js

```diff
@@ -81,7 +105,28 @@ docWarrior.prototype._returnInOrder = function(requested, docs){
   requested.forEach(function(docType){
     if(docs[docType] != '') finalContent += docs[docType] + '\n\n';
   });
+
   return finalContent;
 };
 
+docWarrior.prototype._processSubdirectories = function(parentDirectories, callback) {
+  var self = this;
```

Comment:
Would love to dude although out of scope for the current deadline. Definitely something that we can revisit and improve 👍